enterprise/views.py

```diff
@@ -60,8 +60,11 @@
     filter_audit_course_modes,
 )
 
+from six.moves.urllib.parse import urljoin  # pylint: disable=import-error
 
 logger = getLogger(__name__)  # pylint: disable=invalid-name
+LMS_START_PREMIUM_COURSE_FLOW_URL = urljoin(settings.LMS_ROOT_URL, '/verify_student/start-flow/{course_id}/')
```

Comment:
@bradenmacdonald I believe the `/verify_student/start-flow` endpoint is smart enough to do the right thing if the course mode is `no-id-professional`, but we should definitely test.